assets/models/system/facebook-dinov2-base-imagenet1k-1-layer/spec.yaml

```diff
@@ -0,0 +1,123 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: facebook-dinov2-base-imagenet1k-1-layer
```

Comment:
Why does this name and container path differ?

Reply:
Thanks for catching. Updated.